DQMOffline/Trigger/python/HigPhotonJetHLTOfflineSource_cfi.py

```diff
@@ -28,7 +28,7 @@
     dirname = cms.untracked.string("HLT/Higgs/PhotonJet"), 
     verbose = cms.untracked.bool(False), # default: False
     triggerAccept = cms.untracked.bool(True), # default: True 
-    triggerResultsToken = cms.InputTag("TriggerResults"),
+    triggerResultsToken = cms.InputTag("TriggerResults","","HLT"),
```

Comment:
hi @Dr15Jones , I either do not understand or disagree with the fix.

Are you saying that the framework no longer support reading data from a previous process, unless the process name is given explicitly ?